cmake/bob_robotics.cmake

```diff
@@ -138,12 +116,15 @@ macro(BoB_project)
         endif()
     endif()
     string(TOUPPER ${ROBOT_TYPE} ROBOT_TYPE_UPPER)
+    string(REGEX REPLACE :: _ ROBOT_TYPE_UPPER ${ROBOT_TYPE_UPPER})
```

Comment:
I think I get the logic here but some comments would be nice

Reply:
Good point. This is a bit opaque...